j2/rules/govbr_controle_de_solicitacoes.json

```diff
@@ -0,0 +1,219 @@
+{
```

Comment:
Why did we create a new file for this?  It looks like a copy/paste job from the other bb_xxxx!   Please try to reuse these rules as much as possible